src/assets/CoffeeScriptAsset.js

```diff
@@ -0,0 +1,22 @@
+const JSAsset = require('./JSAsset');
+const config = require('../utils/config');
+const localRequire = require('../utils/localRequire');
+
+class CoffeeScriptAsset extends JSAsset {
+  async transform() {
+    super.transform();
+
+    await this.parseIfNeeded();
+    this.isAstDirty = true;
+  }
```

Comment:
This shouldn't be necessary, after the TypeScript PR this has been fixed by @devongovett in a better way to parse every non js file automatically. This would also break Uglify support.